src/main/app/Resources/modules/data/types/fields/modals/parameters/components/parameters.jsx

```diff
@@ -55,6 +63,16 @@ class ParametersModal extends Component {
     })
   }
 
+  updateConditions(conditionName, conditionValue) {
+    const newConditions = merge({}, this.state.conditions)
+
+    set(newConditions, conditionName, conditionValue)
+
+    this.setState({
```

Comment:
You don't have to store it in the state as the data are already stored in the form store.